Gemfile.lock

```diff
@@ -448,20 +452,22 @@ GEM
       rspec-support (~> 3.12)
     rspec-rerun (1.1.0)
       rspec (~> 3.0)
-    rspec-support (3.12.0)
-    rubocop (1.52.0)
+    rspec-support (3.12.1)
+    rubocop (1.56.3)
+      base64 (~> 0.1.1)
```

Comment:
Well that's unexpected:

> base64 is not part of the default gems since Ruby 3.4.0. Add it to your Gemfile.
> https://github.com/rubocop/rubocop/commit/edde3ebc746f94ed7fa06feda5964190560b516f

I feel like that's a weird decision to make on Ruby's part 😄

Reply:
I agree, ¯\\_(ツ)_/¯